include/chopper/layout/ibf_query_cost.hpp

```diff
@@ -20,34 +20,44 @@ class ibf_query_cost
     ibf_query_cost & operator=(ibf_query_cost &&) = default;
     ~ibf_query_cost() = default;
 
-    constexpr static double exact(size_t const t_max)
+    static double exact(double const fpr, size_t const t_max)
     {
+        auto it = cost_factor.find(fpr);
+
+        if (it == cost_factor.end())
+            it = find_closest_fpr(fpr);
+
         if (contains(t_max))
-            return cost_factor[position(t_max)];
+            return it->second[position(t_max)];
         else
             throw std::invalid_argument("No exact data available for this t_max.");
     }
```

Comment:
I wouldn't mind removing this; it is never used and probably will never be.
It was only used before we interpolated

Reply:
I'll do that later in a separate commit.